style.css

```diff
@@ -0,0 +1,104 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1c1a19;
+}
+
+a {
+  text-decoration: none;
+}
+  
+li {
+  list-style: none;
+  flex-direction: row;
+}
+ 
+.container {
+  padding-left: 24px;
+  padding-right: 24px;
+}
+  
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: white;
+}
+  
+header a {
+  color: white;
+  text-decoration: none;
+  font-size: 17px;
+}
```

Comment:
- [ **OPTIONAL** ] Kindly make sure you are using the correct values of `font size`, `font weight`, and `line height` as the ones provided from the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) design on your **Logo** to make your project design stick to the template as much as possible.